File: LeetCode/1480.running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
        int i = 0, cSum = 0;
        int len = nums.length;
        
        while (i < len){
            cSum += nums[i];
            nums[i] = cSum;
            i++;
        }
        
        return nums;
    }
}

File: LeetCode/1480.running-sum-of-1d-array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        i = 0
        cSum = 0
        Len = len(nums)
        
        while i < Len:
            cSum += nums[i]
            nums[i] = cSum
            i += 1
        return nums